e2e/playwright/actions/src/tests/share/unshare-file-search-results.spec.ts

```diff
@@ -0,0 +1,210 @@
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Site } from '@alfresco/js-api';
+import { ApiClientFactory, FileActionsApi, NodesApi, SharedLinksApi, SitesApi, Utils, test, timeouts } from '@alfresco/playwright-shared';
+import { expect } from '@playwright/test';
+
+test.describe('Unshare a file from Search Results', () => {
+  const random = Utils.random();
+
+  const username = `user-${random}`;
+  const parent = `parent-${random}`;
+  let parentId: string;
+  const searchRandom = random;
+  let sitesApi: SitesApi;
+
+  let file1Id: string;
+  let file2Id: string;
+  let file3Id: string;
+  let file4Id: string;
+  const file1 = `search-file-${searchRandom}-1.txt`;
+  const file2 = `search-file-${searchRandom}-2.txt`;
+  const file3 = `search-file-${searchRandom}-3.txt`;
+  const file4 = `search-file-${searchRandom}-4.txt`;
+
+  const sitePrivate = `site-private-${random}`;
+
+  let fileSite1Id: string;
+  let fileSite2Id: string;
+  const fileSite1 = `search-file-${searchRandom}-Site1.txt`;
+  const fileSite2 = `search-file-${searchRandom}-Site2.txt`;
+
+  const apiClientFactory = new ApiClientFactory();
+
+  test.beforeAll(async () => {
+    try {
+      test.setTimeout(timeouts.extendedTest);
+      await apiClientFactory.setUpAcaBackend('admin');
+      await apiClientFactory.createUser({ username });
+      const nodesApi = await NodesApi.initialize(username, username);
+      sitesApi = await SitesApi.initialize(username, username);
+      const shareApi = await SharedLinksApi.initialize(username, username);
+      const filesAction = await FileActionsApi.initialize(username, username);
+      parentId = (await nodesApi.createFolder(parent)).entry.id;
+
+      file1Id = (await nodesApi.createFile(file1, parentId)).entry.id;
+      file2Id = (await nodesApi.createFile(file2, parentId)).entry.id;
+      file3Id = (await nodesApi.createFile(file3, parentId)).entry.id;
+      file4Id = (await nodesApi.createFile(file4, parentId)).entry.id;
+
+      await sitesApi.createSite(sitePrivate, Site.VisibilityEnum.PRIVATE);
+      const docLibId = await sitesApi.getDocLibId(sitePrivate);
+
+      fileSite1Id = (await nodesApi.createFile(fileSite1, docLibId)).entry.id;
+      fileSite2Id = (await nodesApi.createFile(fileSite2, docLibId)).entry.id;
```

Comment:
Can probably move these 2 to local consts, as they are not getting reassigned anywhere

Reply:
Never mind, only applies to fileSite1Id. fileSite2Id is being used in tests

Reply:
Updated.